README.md

```diff
@@ -24,3 +24,40 @@ To completely remove the environment run
 ```bash
 conda remove --name gostin --all
 ```
+
+
+## Docker
+
+Build the base image locally:
+
+```
+docker build -t softwareunderground/gostin -f base.dockerfile .
+```
+
+To run that image interactively:
+
+```
+docker run -it softwareunderground/gostin
+```
+
+then you can build the image with Jupyter lab:
+
+```
+docker build -t softwareunderground/gostin-jupyter -f jupyter.dockerfile .
+```
+
+then to run Jupyter Lab:
+
+```
+docker run -p 8889:8889 softwareunderground/gostin-jupyter
+```
+
+Perhaps you want to mount some local notebooks you have when running the
+`softwareunderground/gostin-jupyter` container. You can mount this directory
+by doing:
+
+```
+docker run -v $PWD:/root/gostin/ -p 8889:8889 softwareunderground/gostin-jupyter
```

Comment:
I'd have to check but Im not sure on how well this works on both windows / Mac and Linux from memory it causes errors on one of them, if it is just the current directory I think `.:/root/gostin/` should work fine